### PR TITLE
Use gh-pages dependency to deploy storybook #trivial #globalconfig

### PR DESCRIPTION
**Pull request by siggerzz**
Use `gh-pages` to try and fix the Storybook deploy step in CI.

### CODE REVIEW

**Review by jamieomaguire — DISMISSED** [review]

**Comment by siggerzz** [comment]
Will merge after this is approved. As this is a storybook change, no tests / build etc will run.

**Review by DamianMullins — APPROVED** [review]

**Review by JoshuaNg2332 — APPROVED** [review]

**Pull request merged**
